### DISCUSSION TITLE
Show only differences + export those differences

### DISCUSSION THREAD

**Discussion by Tech4Ever** [WinMerge/winmerge#1226] [Q&A] [answered]
Hi,

I would like to compare two excel files and I would like to show only the highlighted differences. Then, in a next step, I would like to export the output on the right side (the newer Excel file version; the other one is always from the past month). I´m using the Compare Excel Plugin.

Is there a way to achieve that?

**Comment by sdottaka** [comment, marked as answer]
If you set the Diff Context to 0 Lines as shown below, only the differences will be displayed.

![image](https://user-images.githubusercontent.com/98126/156764312-06e46168-2afd-45e8-867d-9db32b8253ec.png)

In this state, focus on the right pane, press Ctrl+A and then Ctrl+C to copy only the differences to the clipboard. This can then be pasted into a text editor and saved.

You can also output the differences to a file by selecting the "Tools → Generate Report" or "Generate Patch" menu item.

**Reply by Tech4Ever** [reply]
Perfect, exactly what I was looking for! Thank you!!

**Reply by Andrej730** [reply]
Hi! I'm sorry if I'm missing something obvious but for me Ctrl-D seems to hide different lines instead of showing just them.

Left:
```
CompositorNodeCornerPin -> CORNERPIN
CompositorNodeCrop -> CROP
CompositorNodeCryptomatte -> CRYPTOMATTE
CompositorNodeCryptomatteV2 -> CRYPTOMATTE_V2
CompositorNodeCurveRGB -> CURVE_RGB
CompositorNodeCurveVec -> CURVE_VEC
Node <class 'bpy.types.CompositorNodeCustomGroup'> was skipped
CompositorNodeDBlur -> DBLUR
CompositorNodeDefocus -> DEFOCUS
CompositorNodeDenoise -> DENOISE
CompositorNodeDespeckle -> DESPECKLE
CompositorNodeDiffMatte -> DIFF_MATTE
CompositorNodeDilateErode -> DILATEERODE
CompositorNodeDisplace -> DISPLACE
CompositorNodeDistanceMatte -> DISTANCE_MATTE
CompositorNodeDoubleEdgeMask -> DOUBLEEDGEMASK
CompositorNodeEllipseMask -> ELLIPSEMASK
CompositorNodeExposure -> EXPOSURE
CompositorNodeFilter -> FILTER
CompositorNodeFlip -> FLIP
CompositorNodeGamma -> GAMMA
```

Right:
```
CompositorNodeCornerPin -> CORNERPIN
CompositorNodeCrop -> CROP
CompositorNodeCryptomatte -> CRYPTOMATTE
CompositorNodeCryptomatteV2 -> CRYPTOMATTE
CompositorNodeCurveRGB -> CURVE_RGB
CompositorNodeCurveVec -> CURVE_VEC
Node <class 'bpy.types.CompositorNodeCustomGroup'> was skipped
CompositorNodeDBlur -> DBLUR
CompositorNodeDefocus -> DEFOCUS
CompositorNodeDenoise -> DENOISE
CompositorNodeDespeckle -> DESPECKLE
CompositorNodeDiffMatte -> DIFF_MATTE
CompositorNodeDilateErode -> DILATE_ERODE
CompositorNodeDisplace -> DISPLACE
CompositorNodeDistanceMatte -> DIST_MATTE
CompositorNodeDoubleEdgeMask -> DOUBLE_EDGE_MASK
CompositorNodeEllipseMask -> ELLIPSEMASK
CompositorNodeExposure -> EXPOSURE
CompositorNodeFilter -> FILTER
CompositorNodeFlip -> FLIP
CompositorNodeGamma -> GAMMA
```


https://github.com/user-attachments/assets/b5ef7276-fa0a-446b-887b-edec1c436c77

**Reply by sdottaka** [reply]
If the View → Diff Context → Invert (Hide Different Lines) menu item is checked, uncheck it.

![image](https://github.com/user-attachments/assets/cfd07a3f-750f-4b44-8aa2-fce521df2b2b)

**Reply by Andrej730** [reply]
Thank you!
